test/dirichlet.jl

```diff
@@ -127,3 +128,54 @@ end
     @test entropy(Dirichlet(N, 1)) ≈ -loggamma(N)
     @test entropy(Dirichlet(ones(N))) ≈ -loggamma(N)
 end
+
+@testset "Dirichlet differentiation $n" for n in (2, 10)
+    alpha = rand(n)
+    Δalpha = randn(n)
+    d, ∂d = ChainRulesCore.frule((nothing, Δalpha), Dirichlet, alpha)
+    ChainRulesTestUtils.test_frule(Dirichlet ⊢ ChainRulesCore.NoTangent(), alpha ⊢ Δalpha)
+    _, dp = ChainRulesCore.rrule(Dirichlet, alpha)
+    ChainRulesTestUtils.test_rrule(Dirichlet{Float64} ⊢ ChainRulesCore.NoTangent(), alpha)
```

Comment:
Same here, I assume `NoTangent` is the default:
```suggestion
    ChainRulesTestUtils.test_rrule(Dirichlet{Float64}, alpha)
```